src/user/migrations/0001_initial.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.2.7 on 2021-10-07 04:36
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
```

Comment:
@rodelletran 
Generated migration files should not be pushed.
Please remove all migration files except `/migrations/__init__.py` from this PR.
Then add information in .gitignore to not include in git changes